core/tests/protractor_utils/waitFor.js

```diff
@@ -172,3 +172,5 @@ exports.invisibilityOfInfoToast = invisibilityOfInfoToast;
 exports.visibilityOfInfoToast = visibilityOfInfoToast;
 exports.visibilityOfSuccessToast = visibilityOfSuccessToast;
 exports.modalPopupToAppear = modalPopupToAppear;
+exports.until = until;
+exports.DEFAULT_WAIT_TIME_MSECS = DEFAULT_WAIT_TIME_MSECS;
```

Comment:
We don't need to export these anymore do we?

Reply:
Yup, removed.